src/Http/Http.Extensions/src/RequestDelegateFactory.cs

```diff
@@ -342,6 +349,17 @@ private static Expression CreateArgument(ParameterInfo parameter, FactoryContext
             factoryContext.TrackedParameters.Add(parameter.Name, RequestDelegateFactoryConstants.RouteOrQueryStringParameter);
             return BindParameterFromRouteValueOrQueryString(parameter, parameter.Name, factoryContext);
         }
+        else if (parameter.ParameterType.IsArray && factoryContext.DisableInferredFromBody && ParameterBindingMethodCache.HasTryParseMethod(parameter.ParameterType.GetElementType()!))
```

Comment:
Is `string[]` handled elsewhere? Since `string` doesn't have a `TryParse` it would skip this `if`

Reply:
Haven't done it yet.

Reply:
Done now!